src/visitor.rs

```diff
@@ -60,21 +60,20 @@ pub struct FmtVisitor<'a> {
 }
 
 impl<'a> FmtVisitor<'a> {
+    // Return true if the given span does not intersect with file lines.
+    fn skip_span(&self, span: Span) -> bool {
```

Comment:
We could, though we need to call `get_context()` everytime, which is a bit unfortunate.